api/cas-server-core-api-events/src/main/java/org/apereo/cas/support/events/authentication/adaptive/CasRiskyAuthenticationDetectedEvent.java

```diff
@@ -28,12 +30,16 @@ public class CasRiskyAuthenticationDetectedEvent extends AbstractCasEvent {
 
     private final Object score;
 
+    private final ClientInfo clientInfo;
+
     public CasRiskyAuthenticationDetectedEvent(final Object source, final Authentication authentication,
                                                final RegisteredService service,
-                                               final Object riskScore) {
-        super(source);
+                                               final Object riskScore,
+                                               final ClientInfo clientInfo) {
+        super(source, clientInfo);
         this.authentication = authentication;
         this.service = service;
         this.score = riskScore;
+        this.clientInfo = ClientInfoHolder.getClientInfo();
```

Comment:
This should be corrected.

Reply:
this should be corrected now.  Thanks for catching that.